File: Dasar-dasar Pemrograman/Semester 1/Python/PR - Konsonan.py
def hanya_konsonan(param):
    if (type(param) != str):
        return None

    a = ['a', 'i', 'u', 'e', 'o', ' ']

    for i in range(len(a)):
        param = param.replace(a[i], '')

    print(param)

hanya_konsonan('Nurul Fikri')

File: Dasar-dasar Pemrograman/Semester 1/PHP/PR - Konsonan.php
<?php

function konsonan(String $param) {
    $ks = ['a', 'i', 'u', 'e', 'o', ' '];

    for ($i = 0; $i < 6; $i++){
        $param = str_replace($ks[$i], '', $param);
    }

    echo $param;
}

konsonan('Nurul Fikri');
